bioblend/galaxy/tools/__init__.py

```diff
@@ -105,6 +105,14 @@ def requirements(self, tool_id):
         url = self._make_url(tool_id) + '/requirements'
         return self._get(url=url)
 
+    def reload(self, tool_id):
+        """
+        Reload specified tool.
+        This functionality is available only to Galaxy admins.
+        """
+        url = self._make_url(tool_id) + '/reload'
+        return self._get(url=url)
```

Comment:
This should surely be a PUT request. Would you be happy to make a PR to Galaxy to fix this upstream?

Reply:
It should be a PUT, and we don't limit the method type, so this should already work, also against old Galaxy instances. ... the issues is that a PUT/POST and DELETE request without parameters in the response body are failing. That's a bug for which a fix would be difficult to backport to all bioblend supported versions and ensure everyone is running with the fix, but this will work:
```suggestion
        return self._put(url=ur, payload={}l)
```

I assume that's why other methods in this class are also setting `payload={}l`

Reply:
Actually, it looks like bioblend somehow "smuggles" in a `b"null"` in the request body ... I'm not sure if that is valid or not ... probably not ?. But I guess we don't need to fix that right away given the workaround works.